<ui version="4.0" >
 <class>StringSelector</class>
 <widget class="QWidget" name="StringSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>25</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="whatsThis" >
   <string>Select a string to utilize.  If you wish to create a new string, select the icon to the right.</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="3" >
    <widget class="QToolButton" name="_editString" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Edit selected string.</string>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="Kst::ComboBox" name="_string" />
   </item>
   <item row="0" column="2" >
    <widget class="QToolButton" name="_newString" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Create a new string.</string>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="_selectString" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_string</tabstop>
  <tabstop>_selectString</tabstop>
  <tabstop>_newString</tabstop>
  <tabstop>_editString</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
